heroku.md

```diff
@@ -223,15 +223,22 @@ registration:
 
 CNAME: [YOUR_DESIRED_REGISTRATION].mit.edu
 Target: [YOUR_DESIRED_REGISTRATION].mit.edu.herokudns.com
+Account number: please include our standard billing account number
```

Comment:
Is this the actual text that should appear here?

Reply:
Yeah. Andy said we needed to include one which they have and he kept forgetting to include it so this note will remind them to do so. I didn't want to include the actual account number as we weren't sure if they were considered secret.